### PR TITLE
Motr Provisioner integration issue resolved

### PR DESCRIPTION
**Pull request by VinothVijayakumar-dev**
Signed-off-by: Atul Deshmukh <atul.deshmukh@seagate.com>

# Problem Statement
- Storage_node keys were expected for control_node while running fsm service

# Design
- fixed the values to fetch appropriate type matching its machine-id from conf store. 

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] cortx_setup cluster bootstrap -c yaml:///etc/cortx/cluster.conf - tested for control node and storage node 
- [ ] Test Cases cover Happy Path

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by yeshpal-jain-seagate — APPROVED** [review]

**Review by atulsdeshmukh2312 — APPROVED** [review]

**Review by madhavemuri — APPROVED** [review]

**Review by ivan-alekhin — APPROVED** [review]
Adding +1 as per discussion from Vinoth. For some reason, the patch cannot be merged without +1 from a code owner, and for some reason GH shows that Anatoliy and I are the only code owners in the list. Since there are 3 +1s already, I believe it is fine if I add +1 without digging into the details.

**Comment by ivan-alekhin** [comment]
@VinothVijayakumar-dev, I believe the problem is related to the fact that you don't have this commit (https://github.com/Seagate/cortx-motr/commit/b327aec77b09d7f720b8713e692f176af4753711) somewhere in the source or the target branch (I cannot tell for sure). If you don't want this problem to happen again, please update the codeowners list. Another solution would be to just perform local merges outside of GH UI.

**Pull request merged**
